README.md

```diff
@@ -13,6 +13,8 @@ $ pip3 install bugswarm-client
 ## Usage
 See [this documentation page](http://www.bugswarm.org/docs/toolset/bugswarm-cli).
 
+Please note that artifacts are pulled from ```bugswarm/cached-images```.
```

Comment:
Change this to: Please note that artifacts are first pulled from ```bugswarm/cached-images``` and if not found then they will be pulled from ```bugswarm/images```.